fretboard/src/main/java/mozilla/components/service/fretboard/Experiment.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard
+
+/**
+ * Represents an A/B test experiment,
+ * independent of the underlying
+ * storage mechanism
+ */
+data class Experiment(
+    val id: String,
+    val name: String,
+    val description: String,
+    val language: String,
+    val appId: String,
+    val regions: List<String>,
+    val maxBucket: Int,
+    val minBucket: Int,
+    val last_modified: Long
+)
```

Comment:
Not all properties may be set. You may want to make them _nullable_.

I also wonder if we should introduce more structure here and at least group the "matchers" into a sub-object?